Mathlib/Data/Real/Sqrt.lean

```diff
@@ -440,4 +440,6 @@ lemma sum_sqrt_mul_sqrt_le (s : Finset ι) (hf : ∀ i, 0 ≤ f i) (hg : ∀ i,
     ∑ i ∈ s, √(f i) * √(g i) ≤ √(∑ i ∈ s, f i) * √(∑ i ∈ s, g i) := by
   simpa [*] using sum_mul_le_sqrt_mul_sqrt _ (fun x ↦ √(f x)) (fun x ↦ √(g x))
 
+attribute [irreducible] sqrt
```

Comment:
Can you not put this on the initial `def`?

Reply:
When I do this, I get errors on some of the subsequent `theorem`s. It can be moved to after `sqrt_nonneg`, though.

Alternatively, one can make the definition of `sqrt` irreduciible and  `unseal` it in `continuous_sqrt` and `sqrt_nonneg`.

Should I do one of these, and if so, what is better?

Reply:
I would expect it to work fine on `def sqrt` after adding a few `rw [sqrt]`s where previously we relied on unfolding